proposals/lip-0063.md

```diff
@@ -541,7 +541,7 @@ def createStakersArray():
 def createGenesisDataObj():
     genesisDataObj = {}
     genesisDataObj.initRounds = POS_INIT_ROUNDS
-    r = ceiling((a.header.height - HEIGHT_PREVIOUS_SNAPSHOT_BLOCK) / ROUND_LENGTH)
+    r = ((a.header.height - HEIGHT_PREVIOUS_SNAPSHOT_BLOCK) + ROUND_LENGTH - 1) // ROUND_LENGTH
```

Comment:
As it is not obvious that this expression computes the ceiling, I would introduce a dedicated function for it similar to LIP 57 https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#ceiling.